### PR TITLE
Add PHPDoc & log when module switch the current theme

### PR DESCRIPTION
**Pull request by Quetzacoalt91**

### CODE REVIEW

File: classes/Upgrader.php [hunk 1]
```diff
@@ -351,7 +351,7 @@ public function getXmlMd5File($version, $refresh = false)
      * returns an array of files which are present in PrestaShop version $version and has been modified
      * in the current filesystem.
      *
-     * @return array of string> array of filepath
+     * @return array|false
```
Comment (Quetzacoalt91) [inline classes/Upgrader.php]:
```suggestion
     * @return string[]|false
```

Comment (PierreRambaud) [reply classes/Upgrader.php]: So we must prefer this syntax when we know it's only composed of strings?

Comment (Quetzacoalt91) [reply classes/Upgrader.php]: If we confirm that our analysis tools are able to understand these types, I'm in favor of using them. :)

**Comment by Quetzacoalt91** [comment]
Thank you for review!

**Pull request merged**